src/libraries/System.IO.Pipelines/src/System/IO/Pipelines/PipeReader.cs

```diff
@@ -36,6 +36,8 @@ public abstract partial class PipeReader
         /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see langword="default" />.</param>
         /// <returns>A <see cref="System.Threading.Tasks.ValueTask{T}" /> representing the asynchronous read operation.</returns>
         /// <remarks>The call returns if the <see cref="System.IO.Pipelines.PipeReader" /> has read the minimumLength specified, or is cancelled or completed.</remarks>
+        /// <remarks>Passing a value of 0 for <paramref name="minimumSize" /> will return a <see cref="System.Threading.Tasks.ValueTask{T}" /> that will not complete until
+        /// further data is available. You should instead call <see cref="System.IO.Pipelines.PipeReader.TryRead(System.IO.Pipelines.ReadResult)" /> to avoid a blocking call.</remarks>
```

Comment:
I'm not sure if multiple remarks are allowed. Hover your cursor over the API and check whether both show up in intellisense.

If not supported, you can add paragraphs to the remarks for each topic.

Reply:
Huh, for some reason I thought they were. Updated